coralproject/talk
# [CORL-1126] Run regex in a sandbox so it can time out

## What does this PR do?

Isolates the wordList RegEx inside of a sandbox container so that it can be timed out when it runs for too long.

When it runs for too long, we kill it and flag it with `SYSTEM_WITHHELD`.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Set some banned/suspect words in admin config
- Create some comments with those words
- Create comments without those words
- See the right comments are flagged for review in the moderation queues
